### DISCUSSION TITLE
SignalR vs FluidFramework

### DISCUSSION THREAD

**Discussion by devmanlab** [microsoft/FluidFramework#16137] [General]
I have a scenario:
1. I have near real-time personal notifications to the SPA Typescript/React app
2. In addition I have org-wide notifications
3. Notifications that are org-wide are retained so every person can view them after login

We are using SignlaR but setup is cumbersome, we have persistent storage for the org-wide notifications. They are retained for specified period.

It seems that FluidFramework may solve same problem. Is it just for collaborative data?

**Comment by kashms** [comment]
Hey @Damag3d, you can use Fluid for this scenario.
While the Fluid platform is intended for collaboration between multiple users reading and writing to the same data set, it is perfectly capable of scenarios where most users are simply reading the data.
